### PR TITLE
Add 1-6 howeworks

### PR DESCRIPTION
**Pull request by juliaozm**

### CODE REVIEW

File: 4. strings_and_loops/1. ascii_unicode_converter.py [hunk 1]
```diff
@@ -0,0 +1,40 @@
...
+purpose = input("Choose method (encrypt/decrypt): ")
+count = int(input(f"Number of messages to {purpose}: "))
+
+for indexCount in range(1, count + 1):
+    message = input(f"Enter Message {indexCount}: ")
+
+    if purpose == "encrypt":
+        enctypted_message = []
+        for char in message:
+            enctypted_message.append(ord(char))
```
Comment (samvel-aivazian) [inline 4. strings_and_loops/1. ascii_unicode_converter.py]: `encrypted_message`

File: 4. strings_and_loops/1. ascii_unicode_converter.py [hunk 2]
```diff
@@ -0,0 +1,40 @@
...
+        print(f"Encrypted Message {indexCount}: {' '.join(str(x) for x in enctypted_message)}") 
+
+    if purpose == "decrypt": 
+        dectypted_message = []
+        for num in message.split():
+           dectypted_message.append(chr(int(num)))
```
Comment (samvel-aivazian) [inline 4. strings_and_loops/1. ascii_unicode_converter.py]: `decrypted_message`

File: 5. loops/assigment_tasks/4. escape_room.py [hunk 3]
```diff
@@ -0,0 +1,19 @@
...
+
+code = 1431
+
+while True:
+    number = int(input("Enter the 4-digit escape code: "))
+    if number != code:
+        print ("Wrong code, try again.")
+        continue
+    else: 
+        print ("Correct! You've escaped!")
```
Comment (samvel-aivazian) [inline 5. loops/assigment_tasks/4. escape_room.py]: check spaces in `print(`

File: 6. data_structures(list)/7. mini_database.py [hunk 4]
```diff
@@ -0,0 +1,91 @@
...
+            print("Your new age is: ", age)
+
+        else:
+            print("It's a wrong option...")
+
+    elif option == "exit":
+        print("Exiting...")
+        break
+
+
```
Comment (samvel-aivazian) [inline 6. data_structures(list)/7. mini_database.py]: too many indents

**Review by samvel-aivazian — COMMENTED** [review]
Added comments.

File: 4. strings_and_loops/1. ascii_unicode_converter.py (hunk 1 above)
Comment (pro1code1hack) [reply 4. strings_and_loops/1. ascii_unicode_converter.py]: @samvel-aivazian , do not review H/W of other participants, you can comment directly on the PRs related to projects, but not to the H/W reviews. 

Closing MR, @juliaozm to open a new one and re-assign me as a reviewer

**Pull request closed without merging**